javagen/src/main/java/com/azure/autorest/model/clientmodel/ClientMethod.java

```diff
@@ -287,6 +287,20 @@ public void addImportsTo(Set<String> imports, boolean includeImplementationImpor
             if (settings.isContextClientMethodParameter()) {
                 imports.add("com.azure.core.util.Context");
             }
+
+            // Paging
+            if (getMethodPageDetails() != null) {
+                imports.add("com.azure.core.http.rest.PagedResponseBase");
+                imports.add("com.azure.core.http.rest.PagedResponse");
+                imports.add("com.azure.core.http.rest.PagedFlux");
+                imports.add("com.azure.core.http.rest.PagedIterable");
+                imports.add("com.google.gson.JsonArray");
```

Comment:
We would want to avoid having a specific json serializer in our dependencies - `azure-core` comes with `jackson` and if we have to `JacksonAdapter` will be the serializer to use.

Reply:
OK. I'll try to use class provided by azure-core.

Reply:
`JacksonAdaptor` forces me to give a type. But I don't want to generate model class from swagger. I just need string of each element in `value` and string of `nextLink`. Should we modify `azure-core` to add this ability?
```
{
  "value": [],
  "nextLink": "",
}
```

Reply:
You can use `Object.class` and then try parse the resulting `Object` var (it should be a `Map<String, Object>`, but do have protection code).